src/patternfly/components/Nav/nav.scss

```diff
@@ -354,389 +235,275 @@
-    &.pf-m-hover,
-    &:hover {
-      --pf-c-nav__horizontal-list-link--Color: var(--pf-c-nav__horizontal-list-link--hover--Color);
-
-      &::after {
-        background-color: var(--pf-c-nav__horizontal-list-link--hover--after--BackgroundColor);
-      }
-    }
-
-    &.pf-m-focus,
-    &:focus {
-      --pf-c-nav__horizontal-list-link--Color: var(--pf-c-nav__horizontal-list-link--focus--Color);
-
-      &::after {
-        background-color: var(--pf-c-nav__horizontal-list-link--focus--after--BackgroundColor);
-      }
-    }
-
-    &.pf-m-active,
-    &:active {
-      --pf-c-nav__horizontal-list-link--Color: var(--pf-c-nav__horizontal-list-link--active--Color);
-
-      &::after {
-        background-color: var(--pf-c-nav__horizontal-list-link--active--after--BackgroundColor);
-      }
-    }
-
-    &.pf-m-current {
-      --pf-c-nav__horizontal-list-link--Color: var(--pf-c-nav__horizontal-list-link--m-current--Color);
-
-      &::after {
-        background-color: var(--pf-c-nav__horizontal-list-link--m-current--after--BackgroundColor);
-      }
-    }
-
-    // After - border bottom treatment
-    &::after {
-      position: absolute;
-      bottom: 0;
-      left: 0;
-      display: block;
-      width: 100%;
-      height: var(--pf-c-nav__horizontal-list-link--after--Height);
-      content: "";
-    }
+  --pf-c-nav__link--PaddingTop: var(--pf-c-nav__horizontal-list__link--PaddingTop);
+  --pf-c-nav__link--PaddingRight: var(--pf-c-nav__horizontal-list__link--PaddingRight);
+  --pf-c-nav__link--PaddingBottom: var(--pf-c-nav__horizontal-list__link--PaddingBottom);
+  --pf-c-nav__link--PaddingLeft: var(--pf-c-nav__horizontal-list__link--PaddingLeft);
+  --pf-c-nav__link--Color: var(--pf-c-nav__horizontal-list__link--Color);
+  --pf-c-nav__link--hover--Color: var(--pf-c-nav__horizontal-list__link--hover--Color);
+  --pf-c-nav__link--active--Color: var(--pf-c-nav__horizontal-list__link--active--Color);
+  --pf-c-nav__link--focus--Color: var(--pf-c-nav__horizontal-list__link--focus--Color);
+  --pf-c-nav__link--BackgroundColor: var(--pf-c-nav__horizontal-list__link--BackgroundColor);
+  --pf-c-nav__link--hover--BackgroundColor: var(--pf-c-nav__horizontal-list__link--hover--BackgroundColor);
+  --pf-c-nav__link--active--BackgroundColor: var(--pf-c-nav__horizontal-list__link--hover--BackgroundColor);
+  --pf-c-nav__link--focus--BackgroundColor: var(--pf-c-nav__horizontal-list__link--hover--BackgroundColor);
+  --pf-c-nav__link--m-current--BackgroundColor: var(--pf-c-nav__horizontal-list__link--m-current--BackgroundColor);
+  --pf-c-nav__link--m-current--Color: var(--pf-c-nav__horizontal-list__link--m-current--Color);
+  --pf-c-nav__link--before--BorderColor: var(--pf-c-nav__horizontal-list__link--before--BorderColor);
+
+  .pf-c-nav__link:hover,
+  .pf-c-nav__link:focus,
+  .pf-c-nav__link:active,
+  .pf-c-nav__link.pf-m-current {
+    --pf-c-nav__link--before--BorderBottomWidth: var(--pf-c-nav__horizontal-list__link--before--BorderWidth);
```

Comment:
Same here - what do you think about separating them out? It's less code to write this way and easier to theme, but less customizable and introduces a breaking change if we choose to style one differently. I think it's fair to expect we might style one differently as the simple nav does that.

Reply:
Same with tertiary